ozone_concentrations/create_artifact.jl

```diff
@@ -0,0 +1,81 @@
+using Dates
+using Downloads
+using NCDatasets
+
+using ClimaArtifactsHelper
+
+FILE_URLs = [
+    "https://caltech.box.com/shared/static/hj2yucye0u3l8e4x8o1d0w35ijj5amh6.nc",
+    "https://caltech.box.com/shared/static/h3c8rlpctxpq3uh1r01zhips426lgi0e.nc",
+            ]
+FILE_PATHs = [
+    "vmro3_input4MIPs_ozone_CMIP_UReading-CCMI-1-0_gn_195001-199912.nc",
+    "vmro3_input4MIPs_ozone_CMIP_UReading-CCMI-1-0_gn_200001-201412.nc",
+             ]
+
+const H_EARTH = 7000.0
+const P0 = 1e5
+const HPA_TO_PA = 100.0
+
+Plvl_inv(P) = -H_EARTH * log(P / P0)
+
+function create_ozone(infile_paths, outfile_path)
+    FT = Float64
```

Comment:
Do we need Float64? The file is pretty big.

Reply:
There is no engineering restriction coming from the float type of this file. If Float32 is good for science, you can go for it.